src/containers/PageBuilder/Primitives/SearchCTA/SearchCTA.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Form as FinalForm } from 'react-final-form';
+import { useHistory } from 'react-router-dom';
+import { createResourceLocatorString } from '../../../../util/routes';
+import { useRouteConfiguration } from '../../../../context/routeConfigurationContext';
+import { useConfiguration } from '../../../../context/configurationContext';
+import { FormattedMessage } from '../../../../util/reactIntl';
+
+import FilterCategories from './FilterCategories';
+import FilterDateRange from './FilterDateRange';
+import FilterLocation from './FilterLocation';
+import FilterKeyword from './FilterKeyword';
+import { Button, Form } from '../../../../components';
```

Comment:
This should be before local files. I.e. import first shared components and only after that component from the same directory.

---

How is this affecting the created code chunks? How the size of the code chunks change between this branch and main branch? 
(I've tried to keep the PageBuilder stuff quite independent from the rest of the codebase (to avoid the need to download huge code chunks on the landing page. However, it might be that that battle is already lost because of topbar.)